app/lib/storage/repos/certificate-template.js

```diff
@@ -0,0 +1,60 @@
+const { blobServiceClient } = require('../get-blob-client')
+const { storageConfig } = require('../../../config')
+
+const template = {}
+let logo
+let signature
+
+const getCertificateTemplate = async (exemptionOrder) => {
+  if (!(template[exemptionOrder] && logo && signature)) {
+    template[exemptionOrder] = await getTemplateFile(exemptionOrder)
+    logo = await getStaticFile('logo.png')
+    signature = await getStaticFile('signature.png')
+  }
+
+  return {
+    template: JSON.parse(template[exemptionOrder]),
+    logo,
+    signature
+  }
+}
+
+const getTemplateFile = async (exemptionOrder) => {
+  const container = blobServiceClient.getContainerClient(storageConfig.certificateTemplateContainer)
+
+  await container.createIfNotExists()
```

Comment:
Presumably this needs pre-creating as you have to put a template file into it before use ?? Therefore, createIfNotExists() seems unnecessary, unless you want to run this to fail yet create the container